modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -85,3 +186,262 @@ For further information, see the Kubernetes documentation on https://kubernetes.
+It is important that both `dbms.cluster.discovery.endpoints` and `dbms.cluster.discovery.v2.endpoints` are set during the operation.
+For more information, see <<clustering-discovery-list>>.
+
+* If `dbms.cluster.discovery.resolver_type=DNS`, set `dbms.cluster.discovery.v2.endpoints` to a single domain name and the cluster port.
+Alternatively, if `dbms.cluster.discovery.resolver_type=SRV`, set `dbms.cluster.discovery.v2.endpoints` to a single domain name and the port set to `0`.
+It is important that both `dbms.cluster.discovery.endpoints` and `dbms.cluster.discovery.v2.endpoints` are set during the operation.
+For more information, see <<clustering-discovery-dns>>.
+
+* If `dbms.cluster.discovery.resolver_type=K8S`, set `dbms.kubernetes.discovery.v2.service_port_name` to the name of the service port used in the Kubernetes service definition for the Core's cluster port.
+It is important that both `dbms.kubernetes.service_port_name` and `dbms.kubernetes.discovery.v2.service_port_name` are during the operation.
+For more information, see <<clustering-discovery-k8s>>.
+
+[[discovery-v1-to-v2-in-place]]
+=== In-place rolling
+
+[IMPORTANT]
+====
+In-place rolling reduces fault tolerance temporarily because you are restarting a running server.
+To keep fault-tolerance, you can introduce a fourth server temporarily.
+====
+. Ensure that the new settings are added to the configuration as detailed in the preparation section.
+. Restart server01 with the new setting `dbms.cluster.discovery.version=V1_OVER_V2`.
++
+image:discovery-service/in-place-1-v1-over-v2.png[]
+
+. Then repeat step 2 for server02 and server03.
+Ensure to restart them sequentially, not in parallel.
++
+image:discovery-service/in-place-23-v1-over-v2.png[]
+
+. Using `bolt://`, connect to servers 1, 2, 3, and run the following procedure.
+It is important to connect via `bolt://` because otherwise the procedure might be routed and executed not on the intended server.
++
+[source,cypher]
+----
+CALL dbms.cluster.showParallelDiscoveryState();
+----
++
+They should display "Matching" in the `stateComparison`.
+If they are not, wait and try again.
++
+. Restart server1 again with the new setting `dbms.cluster.discovery.version=V2_OVER_V1`.
++
+image:discovery-service/in-place-1-v2-over-v1.png[]
+
+. Then repeat step 5 for servers 2 and 3.
+Ensure to restart them sequentially, not in parallel.
++
+image:discovery-service/in-place-23-v2-over-v1.png[]
+
+. Similar to step 4, verify that `stateComparison` shows `Matching`.
+
+. Repeat steps 5, 6 and 7, restarting servers 1, 2, and 3 sequentially, with the new setting `dbms.cluster.discovery.version=V2_ONLY`
+image:discovery-service/in-place-123-v2-only.png[]
+. Verify that `CALL dbms.cluster.showParallelDiscoveryState()` now shows `V2_ONLY` running.
+[[discovery-v1-to-v2-new-server]]
+=== New server rolling
```

Comment:
This title doesn't seem to be showing up in the doc: https://neo4j-docs-operations-1739.surge.sh/operations-manual/5/clustering/setup/discovery/